Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,323 @@
+// Carbine
+#define CARBINE RIFLE
+#define CARBINE_MAG RIFLE_MAG
+// AR
+#define AR "CUP_arifle_RPK74M"
+#define AR_MAG "CUP_75Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M:6"
+// Recon Rifle Attachments
+#define RECONRIFLE_ATTACHMENTS  "rhs_acc_tgpa","rhs_acc_2dpZenit"
+// AT
+#define AT "rhs_weap_rpg7"
+#define AT_MAG "rhs_rpg7_PG7VL_mag:3","rhs_rpg7_OG7V_mag:1"
+// MMG
+#define MMG "rhs_weap_pkm"
+#define MMG_MAG "rhs_100Rnd_762x54mmR_green:5"
+// MAT
+#define MAT "rhs_weap_rpg7"
+#define MAT_MAG "rhs_rpg7_PG7VR_mag:2","rhs_rpg7_PG7VL_mag:1","rhs_rpg7_OG7V_mag:1"
+#define MAT_MAG2 "rhs_rpg7_PG7VR_mag:2","rhs_rpg7_OG7V_mag:1"
+#define MAT_OPTIC "rhs_acc_pgo7v"
+// SAM
+#define SAM "rhs_weap_igla"
+#define SAM_MAG "rhs_mag_9k38_rocket:3"
+#define SAM_MAG2 "rhs_mag_9k38_rocket:2"
+// Marksman Rifle
+#define MARKSMAN "rhs_weap_svds"
+#define MARKSMAN_MAG "rhs_10Rnd_762x54mmR_7N1:8"
+// Sniper Rifle
+#define SNIPER "rhs_weap_svds"
+#define SNIPER_MAG "rhs_10Rnd_762x54mmR_7N1:8"
+// Spotter Rifle
+#define SPOTTER "rhs_weap_svds"
+#define SPOTTER_MAG "rhs_10Rnd_762x54mmR_7N1:8"
+// SMG
+#define SMG "rhs_weap_aks74u"
+#define SMG_MAG "rhs_30Rnd_545x39_AK:6"
+// Pistol
+#define PISTOL "rhs_weap_pya"
+#define PISTOL_MAG "rhs_mag_9x19_17:3"
+// Grenades
+#define BASE_FRAG "rhs_mag_rgd5:2"
+#define BASE_GRENADES BASE_FRAG,BASE_SMOKES
+#define LEADER_GRENADES BASE_FRAG,LEADER_SMOKES,"Chemlight_red:2"
+// Gear
+#define BASE_TOOLS COMMON_TOOLS
+#define LEADER_TOOLS COMMON_LEADER_TOOLS,KEY_EAST
+#define BASE_LINKED COMMON_LINKED
+#define LEADER_LINKED COMMON_LEADER_LINKED
+
+class Car {
+  TransportWeapons[] = {AT};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Tank {
+  TransportWeapons[] = {AT};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Helicopter {
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Plane {};
+class Ship_F {};
+
+class potato_msv_rifleman { // rifleman
+  uniform[] = {"rhs_uniform_emr_patchless"};
+  vest[] = {"rhs_6b23_digi_6sh92_vog"};
+  headgear[] = {"rhs_6b27m_digi","rhs_6b27m_digi_bala","rhs_6b27m_digi_ess","rhs_6b27m_digi_ess_bala","rhs_6b47","rhs_6b47_bala","rhs_6b47_ess","rhs_6b47_ess_bala"};
+  backpack[] = {"rhs_assault_umbts"};
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {RIFLE};
+  magazines[] = {RIFLE_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS,"rhs_1PN138"}; // rhs_1PN138 russian single tube NVG
+  linkedItems[] = {BASE_LINKED};
+  attachments[] = {RIFLE_ATTACHMENTS};
+  opticChoices[] = {"rhs_acc_pkas","rhs_acc_pgo7v","rhs_acc_1p29","rhs_acc_1p63","rhs_acc_1p78","rhs_acc_1pn93_2","rhs_acc_ekp1"};
```

Comment:
I'll remove the RPG, that was an accident. Magnified optics are intentional, the choices obviously only appear if the mission maker wants them and the choices serve to give the option between the NVG magnified optics and the non-NVG magnified optics. The problem is there's a few different mag'd optics and I couldn't pick out ones that were comparable to the RCO on the USMC side so I just left them all in for now.